CK2Plus_expanded/localisation/000_CK2Plus_expanded_religions.csv

```diff
@@ -187,6 +187,122 @@ d_keepers_tirnanog_adj;Keeper;c
+
+# Landed Titles
+temple_baron_basque_pagan;Sorgin;;;;;;;;;;;;;x
+temple_baron_basque_pagan_reformed;Sorgin;;;;;;;;;;;;;x
+temple_baron_female_basque_pagan;Sorgin;;;;;;;;;;;;;x
+temple_baron_female_basque_pagan_reformed;Sorgin;;;;;;;;;;;;;x
+
+temple_barony_basque_pagan;Akellare;;;;;;;;;;;;;x
+temple_barony_basque_pagan_reformed;Akellare;;;;;;;;;;;;;x
+temple_barony_of_basque_pagan;Akellare of;Akellare de;Akellare von;;Akellare de;;;;;;;;;x
+temple_barony_of_basque_pagan_reformed;Akellare of;Akellare de;Akellare von;;Akellare de;;;;;;;;;x
+
+temple_count_basque_pagan;Sorgin;;;;;;;;;;;;;x
+temple_count_basque_pagan_reformed;Sorgin;;;;;;;;;;;;;x
+temple_count_female_basque_pagan;Sorgin;;;;;;;;;;;;;x
+temple_count_female_basque_pagan_reformed;Sorgin;;;;;;;;;;;;;x
+
+temple_county_basque_pagan;Tenplu;;;;;;;;;;;;;x
+temple_county_basque_pagan_reformed;Tenplu;;;;;;;;;;;;;x
+temple_county_of_basque_pagan;Tenplu of;Tenplu de;Tenplu von;;Tenplu de;;;;;;;;;x
+temple_county_of_basque_pagan_reformed;Tenplu of;Tenplu de;Tenplu von;;Tenplu de;;;;;;;;;x
+
+temple_duke_basque_pagan;Head Sorgin;;;;;;;;;;;;;x
+temple_duke_basque_pagan_reformed;Head Sorgin;;;;;;;;;;;;;x
+temple_duke_female_basque_pagan;Head Sorgin;;;;;;;;;;;;;x
+temple_duke_female_basque_pagan_reformed;Head Sorgin;;;;;;;;;;;;;x
+
+temple_king_basque_pagan;Great Sorgin;;;;;;;;;;;;;x
+temple_king_basque_pagan_reformed;Great Sorgin;;;;;;;;;;;;;x
+temple_king_female_basque_pagan;Great Sorgin;;;;;;;;;;;;;x
+temple_king_female_basque_pagan_reformed;Great Sorgin;;;;;;;;;;;;;x
+
+temple_kingdom_basque_pagan;Federation;F�d�ration;F�deration;;Federaci�n;;;;;;;;;x
+temple_kingdom_basque_pagan_reformed;Federation;F�d�ration;F�deration;;Federaci�n;;;;;;;;;x
+temple_kingdom_of_basque_pagan;Federation of;F�d�ration de;F�deration von;;Federaci�n de;;;;;;;;;x
+temple_kingdom_of_basque_pagan_reformed;Federation of;F�d�ration de;F�deration von;;Federaci�n de;;;;;;;;;x
+
+temple_empire_basque_pagan;Nation;Nation;Nation;;Naci�n;;;;;;;;;x
+temple_empire_basque_pagan_reformed;Nation;Nation;Nation;;Naci�n;;;;;;;;;x
+temple_empire_of_basque_pagan;Sacred Realm of;Sacr� Domaine de;Sakrales Reich von;;Dominio Sagrado de;;;;;;;;;x
+temple_empire_of_basque_pagan_reformed;Sacred Realm of;Sacr� Domaine de;Sakrales Reich von;;Dominio Sagrado de;;;;;;;;;x
+
+# Artifacts
+scepter_basque_fylkir;TODO;TODO;TODO;;TODO;;;;;;;;;x
+scepter_basque_fylkir_desc;TODO;TODO;TODO;;TODO;;;;;;;;;x
+
+# Triggers
+religion_openly_basque_or_reformed_tooltip;Publicly follows the �Y[basque_pagan.GetName]�! religion;Suit publiquement la religion �Y[basque_pagan.GetName]�!;Folgt �ffentlich der �Y[basque_pagan.GetName]en�! Religion;;Sigue p�blicamente la religi�n �Y[basque_pagan.GetName]�!;;;;;;;;x
+
+# Holy Order
+d_sword_of_ama_lur;Sword of Ama Lur;�p�e de Ama Lur;Schwert von Ama Lur;;Schwert von Ama Lur;;;;;;;;;;;;;x
+d_sword_of_ama_lur;Sword of Ama Lur;�p�e de Ama Lur;Schwert von Ama Lur;;Schwert von Ama Lur;;;;;;;;;;;;;x

```

Comment:
Duplicate of above line. Spanish localisation is German

Reply:
Ah sorry, second should be `d_sword_of_ama_lur_adj`. I couldn't think of a better adjective than just the name of the holy order.